src/js/harness.js

```diff
@@ -20,15 +20,21 @@ fluid.registerNamespace("gpii.test.couchdb.harness");
  */
 gpii.test.couchdb.harness.startIfNeeded = function (that, isCouchUp) {
     var startPromise = fluid.promise();
-    that.worker.isUp().then(function () {
-        if (isCouchUp) {
-            startPromise.resolve("CouchDB is already running, no need to start it.");
-        }
-        else {
-            var innerStartPromise = that.worker.startup();
-            innerStartPromise.then(startPromise.resolve, startPromise.reject);
-        }
-    }, startPromise.reject);
+    // If our worker has already been destroyed, startup is not needed.
+    if (that.worker.lifecycleStatus === "destroyed") {
```

Comment:
A cleaner check is fluid.isDestroyed(that.worker) - the lifecycleStatus flag does not have API status

Reply:
Yup, fixed.